--- {{ lokole_venv }}/lib/python{{ python_ver }}/site-packages/opwen_email_client/webapp/forms/register.py.orig 	2021-05-24 06:30:58.301125145 +0000
+++ {{ lokole_venv }}/lib/python{{ python_ver }}/site-packages/opwen_email_client/webapp/forms/register.py	2021-05-23 23:30:01.826391284 +0000
@@ -35,7 +35,7 @@
         response = post(client_create_url,
                         json={'domain': client_domain},
                         headers={'Authorization': 'Bearer {}'.format(token)})
-        if response.status_code != 200:
+        if response.status_code != 201:
             raise ValidationError(i8n.FAILED_REGISTRATION)
         register.delay(name, token, path)
 
