diff --git a/cmd/courier/main.go b/cmd/courier/main.go
index 83cf2a7f..d2e1ab97 100644
--- a/cmd/courier/main.go
+++ b/cmd/courier/main.go
@@ -71,6 +71,7 @@ import (
 	_ "github.com/nyaruka/courier/handlers/whatsapp_legacy"
 	_ "github.com/nyaruka/courier/handlers/yo"
 	_ "github.com/nyaruka/courier/handlers/zenvia"
+  _ "github.com/nyaruka/courier/handlers/wuzapi"
 	"github.com/nyaruka/courier/runtime"
 
 	// load available backends
@@ -133,3 +134,4 @@ func main() {
 
 	server.Stop()
 }
+// TESTS-PASSED
