--- install_prereq.orig	2023-04-01 01:41:56.859545082 -0500
+++ install_prereq	2023-04-01 01:44:28.744269701 -0500
@@ -193,8 +193,13 @@
 		tocheck="${tocheck} ^${pack}$ ~P^${pack}$"
 	done
 	pkgs=$(aptitude -F '%c %p' search ${tocheck} 2>/dev/null | awk '/^p/{print $2}')
+	arch=$(uname -m)
 	if [ ${#pkgs} -ne 0 ]; then
-		echo $pkgs | sed -r -e "s/ ?[^ :]+:i386//g"
+		if [ "$arch" = "x86_64" ]; then
+			echo $pkgs | sed -r -e "s/ ?[^ :]+:i386//g"
+		elif [ "$arch" = "aarch64" ]; then
+			echo $pkgs | sed -r -e "s/ ?[^ :]+:armhf//g"
+		fi
 	fi
 }
 
